nixos/modules/services/mail/rspamd.nix

```diff
@@ -294,20 +319,44 @@ in
       gid = config.ids.gids.rspamd;
     };
 
-    environment.etc."rspamd.conf".source = rspamdConfFile;
+    services.rspamd = {
+      socketActivation = mkDefault (!opts.bindSocket.isDefined && !opts.bindUISocket.isDefined);
+
+      workers = mkIf cfg.postfix.enable {
+        controller = {};
+        rspamd_proxy = {
+          bindSockets = [ {
+            socket = "/var/lib/postfix/queue/private/rspamd";
+            owner = "rspamd";
+            group = "postfix";
+          } ];
+        };
+      };
+
+      overrides."logging.inc" = mkDefault ''
+        type = "syslog";
+      '';
+    };
+
+    services.postfix.extraConfig = mkIf cfg.postfix.enable ''
+      smtpd_milters = unix:private/rspamd
+      non_smtpd_milters = $smtpd_milters
```

Comment:
The official documentation does not mention this: https://rspamd.com/doc/integration.html

Reply:
No but if you want rspamd to also scan mails from command line you need this. And if you use the DKIM module in rspamd to also add DKIM signatures you want rspamd to also be called for command line mails.

Reply:
makes sense.